slime_environments/environments/SlimeEnvMultiAgent.py

```diff
@@ -154,9 +163,9 @@ def __init__(self,
                 self.coords.append((x, y))  # "centre" of the patch or turtle (also ID of the patch)
 
         pop_tot = self.population + self.learner_population
-        self.agents = [i for i in range(self.population, pop_tot)]  # DOC learning agents IDs
-        self._agent_selector = agent_selector(self.agents)
-        self.agent = self._agent_selector.next()
+        self.possible_agents = ["agent_"+ str(i) for i in range(self.population, pop_tot)]  # DOC learning agents IDs

```

Comment:
Why adding a prefix ("agent_") to the agent name? is it required by pettingzoo API? or is it just for more clarity?

Reply:
As far as I remember, pettingZoo requires 'possible_agents' to be an array of strings.

Reply:
thanks, will double-check but fine anyway :)